backends/occa/ceed-occa-qfunction-op.c

```diff
@@ -152,6 +153,8 @@ int CeedQFunctionAllocOpOut_Occa(CeedQFunction qf, CeedInt Q,
       odx+=1;
       break;
     case CEED_EVAL_GRAD:
+      assert(dim>0);
```

Comment:
Can we produce an error message instead of an `assert`?

Reply:
I think a better fix is to change the initial value above from 0 to 1 and put the error check in the interface.